examples/cron/compound-staking-cron/src/lib.rs

```diff
@@ -77,12 +98,25 @@ pub fn execute(
 pub fn query(_deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
     let mut instructions = vec![];
     // 1. parse claimable rewards
+    let fis = &msg.fis_input[0];
+    let rewards_response = from_json::<RewardsResponse>(fis.data.first().unwrap()).unwrap();
+
+    let rewards = rewards_response.rewards.clone(); 
+
+    let validator_address = rewards[0].validator_address.clone();
```

Comment:
1/ we should consider all `rewards` in the reward array
2/ should return proper error when there is no rewards to claim so that:
- There will be readable error in cron log => easier to debug
- If we fail immediately it will save some gas